bigquery/tests/unit/test_dbapi_cursor.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import unittest
+
+
+class TestCursor(unittest.TestCase):
+
+    @staticmethod
+    def _get_target_class():
+        from google.cloud.bigquery.dbapi import Cursor
+        return Cursor
+
+    def _make_one(self, *args, **kw):
+        return self._get_target_class()(*args, **kw)
+
+    def test_ctor(self):
+        from google.cloud.bigquery.dbapi import connect
+        from google.cloud.bigquery.dbapi import Cursor
+        connection = connect(_Client())
+        cursor = self._make_one(connection)
+        self.assertIsInstance(cursor, Cursor)
+        self.assertIs(cursor.connection, connection)
+
+    def test_close(self):
+        from google.cloud.bigquery.dbapi import connect
+        from google.cloud.bigquery.dbapi import Cursor
+        connection = connect(_Client())
+        cursor = connection.cursor()
+        # close() is a no-op, there is nothing to test.
+        cursor.close()
+
+    def test_fetchone_wo_execute_returns_none(self):
+        from google.cloud.bigquery.dbapi import connect
+        from google.cloud.bigquery.dbapi import Cursor
+        connection = connect(_Client())
+        cursor = connection.cursor()
+        row = cursor.fetchone()
```

Comment:
[PEP 249](https://www.python.org/dev/peps/pep-0249/#fetchone) says this case should raise an `Error`.